14-arrays/exercises/07-assign/main.go

```diff
@@ -42,4 +36,11 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	books := [3]string{"Kafka's Revenge", "Stay Golden", "Everythingship"}
+	lower := [3]string{"kafka's revenge", "stay golden", "everythingship"}
```

Comment:
I think the problem wants you to copy the books array into lower like below
lower = books
then change the titles to lowercase
similarly do the same to uppercase array

Reply:
I got confused, that makes more sense. Fixed